src/lib/containers/home_page/featured-data/FeaturedDataPlots.tsx

```diff
@@ -16,18 +19,30 @@ const FeaturedDataPlots: React.FunctionComponent<FeaturedDataPlotsProps> = props
     sql,
     token,
   } = props
+  // What mode are we in?  Either every card has a different selected facet (requiring a different query),
+  // or we're showing the facet counts for a single query.  This controls the layout, and how the cards are populated.
+  const isQueryPerCard = (configs[0] as any).selectFacetColumnName
   return (
-  <div className="FeaturedDataPlots">
-    {configs.map(config => {
-      return <div className="FeaturedDataPlots__card">
-        <QueryWrapperFacetPlotsCard 
-          {...config}
-          rgbIndex={rgbIndex}
-          sql={sql}
-          token={token} />
-      </div>
-    })}
-  </div>)
+    <div className={`FeaturedDataPlots${isQueryPerCard ? '__queryPerCard' : '__singleQuery'}`}>
+      {configs.map((config:any) => {
+        return <>
+          {isQueryPerCard && 
+            <QueryPerFacetPlotsCard
```

Comment:
This is the old sub-component.  Renamed to differentiate it from the alternative.